.ci/azure/linux.yml

```diff
@@ -26,6 +26,7 @@ jobs:
     BUILD_TYPE: Release
     REPO_DIR: $(Build.Repository.LocalPath)
     OPENVINO_REPO_DIR: $(REPO_DIR)/../openvino
+    OPENVINO_CONTRIB_REPO_DIR: $(REPO_DIR)/../openvino_contrib
```

Comment:
duplication, REPO_DIR should be removed in this case, and use $(Build.Repository.LocalPath)